### ISSUE TITLE
sorry path issues but for noob its very difficult

### ISSUE THREAD

**Issue by sarfrazramay** [pgmodeler/pgmodeler#507]
Hi,
The installation instructions should be in the order which they should be executed, i found them really hard to follow, even then i could not get it to work. I am sorry if this is a repetition but i was appreciate help.

As seen below, i cant get the libxml path right. Do i have to install it or build it or something. i just downloaded the tar from the link you have given and copied the extracted folder in the /usr/lib and /usr/inlcude directories. What am i missing ?

camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ qmake pgmodeler.pro
Project MESSAGE: Warning: unknown QT: widgets
Project MESSAGE: Warning: unknown QT: printsupport
Project ERROR: Package libxml-2.0 not found
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ pkg-config libpq --cflags --libs
-I/usr/include/postgresql  -L/usr/lib/postgresql/[VERSION]/lib/libpq.so -lpq  
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ 
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ pkg-config libxml-2.0 --cflags --libs
Package libxml-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libxml-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libxml-2.0' found
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$ ./start-pgmodeler.sh
./start-pgmodeler.sh: line 29: pgmodeler: command not found
camaraadmin@misserver-dublin:~/pgmodeler-0.7.2$

**Comment by bwakkie** [comment]
If I copy the command "pkg­-config libxml-­2.0 ­--cflags --libs" from the website I get the following ...
`pkg­--config libxml--­2.0 ­---cflags --libs`
... in my terminal. So a couple of '**dashes**' too much, but they aren't _real_ dashes! 
I guess a wrong unicode convertion somewhere.
` pkg-config libxml-2.0 --cflags --libs` does work in my terminal.
